spec/requests/developer_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Developers Endpoint", type: :request do
+  let(:developers) { create_list(:developer, 5) }
+
+  it "GET /developers" do
+    developers
+    get developers_path, as: :json
+    expect(response.parsed_body).to eq(Developer.all.as_json)
+  end
+
+  it "GET /developers/:developer_id" do
+    developer = developers.sample
+    get developer_path(developer), as: :json
+    expect(response.parsed_body).to eq(developer.as_json)
+  end
+
+  it "POST /developers" do
+    developer = attributes_for(:developer)
+
+    post developers_path, params: developer, as: :json
+    expect(response).to have_http_status(:created)
+    expect(response.body).to include(developer[:name])
+  end
+
+  it "PUT /developers" do
+    developer = create(:developer)
+    different_name = "A different developer name"
+    developer.name = different_name
+
+    put developer_path(developer), params: developer.as_json, as: :json
+    expect(response).to have_http_status(:ok)
+    expect(response.parsed_body.except("updated_at")).to eq(developer.as_json.except("updated_at"))
+  end
+
+  it "DELETE /developers/:developer_id" do
+    developer = developers.sample
+    delete developer_path(developer), as: :json
+    expect(response).to have_http_status(:no_content)
```

Comment:
Why no content? You could return the destroyed entity just a though no action needed

Reply:
There doesn't seem to be a hard rule, both 200 and 204 work, and I didn't think there was any more "status" to include in the body?